File: src/main/java/gq/noxiuam/bitches/object/type/special/Gami.java
package gq.noxiuam.bitches.object.type.special;

import gq.noxiuam.bitches.object.type.AnnoyingBitch;
import lombok.Getter;

/**
 * @author Noxiuam
 * https://noxiuam.gq
 */
@Getter
public class Gami extends AnnoyingBitch {

    private final boolean ownsAShitServer;

    public Gami(boolean ownsAShitServer) {
        super(1000, false, true, true);
        this.ownsAShitServer = ownsAShitServer;
        this.build();
    }

}

File: src/main/kotlin/gq/noxiuam/bitches/object/type/special/Gami.kt
package gq.noxiuam.bitches.`object`.type.special

import gq.noxiuam.bitches.`object`.type.AnnoyingBitch
import lombok.Getter

/**
 * @author Noxiuam
 * https://noxiuam.gq
 */
@Getter
class Gami(private val ownsAShitServer: Boolean) : AnnoyingBitch(1000, false, true, true) {
    init {
        this.build()
    }
}